Kyrychenko/quizEntity/src/main/java/service/UserService.java

```diff
@@ -0,0 +1,51 @@
+package service;
+
+import model.User;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class UserService {
+    private List<User> userList;
+    private static volatile UserService instance;
+
+    private UserService() {
+        this.userList = new ArrayList<>();
+    }
+
+    public static UserService getInstance() {
+        UserService localInstance = instance;
+        if (localInstance == null) {
+            synchronized (QuizService.class) {
+                localInstance = instance;
+                if (localInstance == null) {
+                    instance = localInstance = new UserService();
+                }
+            }
+        }
+        return localInstance;
+    }
+
+    public boolean isUserExist(String login, String password) {
+        User temp = new User("null", login, password);
```

Comment:
Why here is "null"? I think in system can be only one user with login, so you can check only login